packages/auth/src/core/auth/register.test.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { expect, use } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import sinonChai from 'sinon-chai';
+import { FirebaseError } from '@firebase/util';
+import { registerAuth } from './register';
+import { ClientPlatform } from '../util/version';
+use(sinonChai);
+use(chaiAsPromised);
+
+describe('it should register successfully for each platform type', () => {
+  it('should not throw error for node platform', () => {
+    expect(() => {
+      registerAuth(ClientPlatform.NODE);
```

Comment:
How does the API Key get propagated in this test?

I see we have an assert to check that API key is non-empty.

Reply:
I removed this as it did not directly test the changes we made.